### ISSUE TITLE
Visualization code

### ISSUE THREAD

**Issue by dilpath** [PEtab-dev/petab_select#9]
- line graph of history of best model criterion at each iteration
- network graph of all visited models

Other suggestions welcome

**Comment by dweindl** [comment]
> Other suggestions welcome

For candidate spaces in the form of https://github.com/PEtab-dev/petab_select/blob/8d97c58e1f4a671fecb19881827db237f12f51e0/test_cases/0009/model_space.tsv: [UpSet plots](https://en.wikipedia.org/wiki/UpSet_Plot)

**Comment by dweindl** [comment]
> it includes some misleading elements by default

right. wasn't too hard to get rid of it though. but yeah, via matplotlib...

**Comment by dilpath** [comment]
Will be closed by #36
